vision/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
-include README.rst LICENSE
+include README.rst LICENSE requirements.txt
 recursive-include google *.json *.proto
```

Comment:
I would like that too, but I assert it is out of scope for _this_. I will see about making that change though.